WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1221,6 +1240,10 @@ public void onResponse(JSONObject response) {
                         @Override
                         public void onErrorResponse(VolleyError error) {
                             if (isAdded()) {
+                                HashMap<String, Object> errorProperty = new HashMap<>();
+                                errorProperty.put(ANALYTICS_ERROR_PROPERTY_KEY, error.getMessage());
```

Comment:
Note that `error.getMessage()` may return null. I'm not sure that would cause problems but wanted to point it out just in case.